packages/core/src/index.ts

```diff
@@ -134,5 +134,6 @@ export { ErrorSource } from './domain/error/error.types'
 export { RawError } from './domain/error/error.types'
 export { RawErrorCause } from './domain/error/error.types'
 export { ErrorWithCause } from './domain/error/error.types'
+export { Csp } from './domain/error/error.types'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏it was already the case but it seems a bit weird to add one export by type, maybe we could enforce the grouping somehow to have:
```js
export { ErrorHandling, ErrorSource, ... } from './domain/error/error.types'
```

Reply:
✅ done! I think this is the kind of things that could be automated. Any thoughts on [eslint/sort-imports](https://eslint.org/docs/latest/rules/sort-imports) rule?